### PR TITLE
Login restrction to xsolve or chilid domain, basic error handling.

### PR DESCRIPTION
**Pull request by bebe84**

### CODE REVIEW

File: src/redux/auth.js [hunk 1]
```diff
@@ -35,4 +35,4 @@ export const login = () => dispatch => dispatch({
     const tokens = await Firebase.login();
     return tokens;
   },
-});
+}).catch(error => errorHelper(error));
```
Comment (mateuszmatyszkowicz) [inline src/redux/auth.js]: Here also could be `catch(errorHelper)` I'm not sure which one notation is commonly used or standard, You are more experienced so, could you tell me?

File: src/helpers/Firebase.js [hunk 2]
```diff
@@ -22,14 +24,21 @@ class Firebase {
...
         scopes,
       });
 
+      const {
+        idToken, accessToken, refreshToken, type,
+      } = auth;
+
       if (type === 'success') {
+        const emailValid = domainRegEx.test(auth.user.email);
+        if (!emailValid) {
```
Comment (mateuszmatyszkowicz) [inline src/helpers/Firebase.js]: Before condition should be empty line for better readability.

File: src/redux/auth.js (hunk 1 above)
Comment (bebe84) [reply src/redux/auth.js]: You mean that I should omit error object here?

File: src/helpers/Firebase.js (hunk 2 above)
Comment (bebe84) [reply src/helpers/Firebase.js]: My linter and config is not letting that happen, I am using very restrictive AirBnB RN config.

Comment (mateuszmatyszkowicz) [reply src/helpers/Firebase.js]: So that's okey. It can stay that to keep consistency.

Comment (bebe84) [reply src/helpers/Firebase.js]: Thanks!

**Pull request merged**